Sources/apollo-ios-pagination/AnyGraphQLPager.swift

```diff
@@ -0,0 +1,92 @@
+import Apollo
+import ApolloAPI
+
+public class AnyGraphQLQueryPager<Model> {
+  public typealias Output = Result<([Model], [[Model]], UpdateSource), Error>
```

Comment:
If the guarantee is that the `Model` is going to be the same for each element from both `InitialQuery` and `NextQuery`, then can't we just make the output here be a single concatenated array of `Model`?

I think it adds some more flexibility here. As if you really do want to do the transforms but also keep page separation, then you can create your own `Page` struct (which could include page index or any other page metadata you want) and then have the `Model` type be your pages. Then you would get an output of `[Page]` with each page including it's models.

But if you don't need that, and you just want to have all of your models concatenated together, then you can just get an array of `[Model]`.